Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-Suspicious_Powershell_Command_Execution.yml

```diff
@@ -0,0 +1,1481 @@
+    type: condition
+    task:
+      id: ba8dc3c7-904a-420a-8d24-4bdd1e259958
+      version: -1
+      name: Check Extracted IOC's Reputation
+      description: Validate reputation of extracted command line indicators for potential threat analysis
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "6"
+      Malicious:
+      - "10"
+    separatecontext: false
+    conditions:
+    - label: Malicious
+      condition:
+      - - operator: isEqualNumber
+          left:
+            value:
+              simple: DBotScore.Score
+            iscontext: true
+          right:
+            value:
+              simple: "3"
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 507,
+          "y": 392
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: b4e2c3d8-bc10-4af9-8455-3a26f2d715c0
+    type: title
+    task:
+      id: b4e2c3d8-bc10-4af9-8455-3a26f2d715c0
+      version: -1
+      name: Investigation
+      description: Checks the file reputation of the specified hash.
+      type: title
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "25"
+      - "30"
+      - "33"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 507,
+          "y": 589
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: true
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: c3242ea7-8b65-4f3d-86f2-49185c62d1b7
+    type: title
+    task:
+      id: c3242ea7-8b65-4f3d-86f2-49185c62d1b7
+      version: -1
+      name: Remediation
+      type: title
+      iscommand: false
+      brand: ""
+      description: ''
+    nexttasks:
+      '#none#':
+      - "11"
+      - "38"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -392,
+          "y": 1581
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "11":
+    id: "11"
+    taskid: e07084eb-7afc-45a1-81dd-8c21920e1f0d
+    type: regular
+    task:
+      id: e07084eb-7afc-45a1-81dd-8c21920e1f0d
+      version: -1
+      name: Terminate Causality
+      description: Terminate a process tree by its causality ID. Available only for Cortex XSIAM 2.4.
+      script: '|||core-terminate-causality'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#error#':
+      - "12"
+    scriptarguments:
+      agent_id:
+        simple: ${alert.agentid}
+      causality_id:
+        simple: ${alert.cid.[0]}
```

Comment:
Why do we need to explicitly use .[0]?

Reply:
without it it returns as an array (e.g. ["Advk5bWRz4EAAC6AAAAAAA=="]) and the task fails